Packs/FortiSIEM/Integrations/FortiSIEM/FortiSIEM.py

```diff
@@ -277,7 +277,7 @@ def GetIncidentsByOrg(queryId):
             m = mm.split("=")[-1]
```

Comment:
```suggestion
            m = int(mm.split("=")[-1])
```
Do it once instead of all the conversions